deploy/helms/legion/templates/jenkins/jenkins-server.yaml

```diff
@@ -1,4 +1,28 @@
 {{- if .Values.jenkins.enabled }}
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: "{{ .Release.Name }}-jenkins-dynamic-scripts"
+data:
+  configure-cloud.groovy: |-
+    import org.csanchez.jenkins.plugins.kubernetes.*
+    import jenkins.model.*
+
+    cloud = new KubernetesCloud(
+        'kubernetes', // name
+        null, // templates
+        null, // server url
+        '{{ .Release.Namespace }}', // namespace
```

Comment:
What is release namespace?

Reply:
It is helm variable - namespace when chart will be deployed